.github/workflows/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: Generate and publish docs
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Generate docs with jazzy and publish to Github pages
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Generate Docs
+        run: |
+          brew install sourcekitten
```

Comment:
Do we want to pin to a version?

Reply:
Unfortunately it's not possible to pin a version with Homebrew, but I think I can adjust the script to use the sourcekitten binary embedded in jazzy and bypass this install entirely.